chain/jsonrpc-primitives/src/types/receipts.rs

```diff
@@ -52,12 +53,13 @@ impl From<near_client_primitives::types::GetReceiptError> for RpcReceiptError {
             near_client_primitives::types::GetReceiptError::UnknownReceipt(hash) => {
                 Self::UnknownReceipt(hash)
             }
-            near_client_primitives::types::GetReceiptError::Unreachable(s) => {
+            near_client_primitives::types::GetReceiptError::Unreachable(error_message) => {
+                error!(target: "jsonrpc", "Unreachable error occurred: {}", &error_message);
```

Comment:
I prefer thinking of the logging levels in the following way:

* error - it is a full-stop, there is a need for manual recovery, we should not progress further
* warning - we want to bring the information to the attention, but the program can keep going
* info - whatever we think a user may want to know during the normal execution
* debug - more details on what is happening
* trace - the level at which we want to know every single meaningful step the program makes

Thus, let's make them `warn` instead of `error`